config/application.rb

```diff
@@ -61,5 +61,8 @@ class Application < Rails::Application
 
     # Default page size for paging results
     config.x.admin.pagination_per_page = 50
+
+    #Onboarding email address for user contact
+    config.x.admin.onboarding_email = 'LAAapplyonboarding@justice.gov.uk'
```

Comment:
What are we thinking with this for different environments?

Reply:
Thinking we shouldn't show a live email address unless it's prod?